README.md

```diff
@@ -0,0 +1,367 @@
+[![codecov](https://codecov.io/gh/tendermint/liquidity/branch/develop/graph/badge.svg)](https://codecov.io/gh/tendermint/liquidity?branch=develop)
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/tendermint/liquidity)](https://pkg.go.dev/github.com/tendermint/liquidity)
+
+# Liquidity Module
+
+The liquidity module serves AMM (Automated Market Makers) style decentralized liquidity providing and coin swap functions.
+
+The module enables users to create a liquidity pool, make deposits and withdrawals, and request coin swaps from the liquidity pool.
+
+This module will be used in the [Cosmos Hub](https://hub.cosmos.network/main/hub-overview/overview.html), and it can be used in any other [Cosmos SDK](https://github.com/cosmos/cosmos-sdk) based blockchain projects.
+
+- The Cosmos Hub AMM should have strong philosophy of inclusiveness of users from different blockchains because its prime utility is inter-blockchain communication.
```

Comment:
@SenorMonito only after updating some of this README did I notice that other Cosmos SDK modules do not have a README. I'll apply these edits in the other docs. 
(and hi!)

Reply:
As of now, I think `README.md` file should be where it is supposed to be because it is the file that explains what this repository is all about when people come into this repository. And, the liquidity module hasn't decided to be one of the Cosmos SDK modules. It needs to be approved by Cosmos Hub governance. Once it is 100% decided, then we can start asking SDK devs for their opinions and suggestions about this.